rfq/order.go

```diff
@@ -297,6 +297,12 @@ func (c *AssetPurchasePolicy) CheckHtlcCompliance(
 		return fmt.Errorf("error summing asset balance: %w", err)
 	}
 
+	// Due to rounding errors, we may slightly underreport the incoming
+	// value of the asset. So we increase it by exactly one asset unit to
+	// ensure that the fee logic in lnd does not reject the HTLC.
```

Comment:
> fee logic in *lnd*

This is a somewhat confusing comment to me. Lnd fee logic doesn't come into play in this compliance check, does it? We check the inboundAmount against the outbound amount a few lines down and proceed accordingly.

Reply:
You're right, this was Copilot suggesting a comment based on other instances in this file :sweat_smile: In some places we report the converted value back to `lnd` and then the fee logic there decides if an HTLC pays enough fees. But not in this case. Updated the comment.